src/test/regress/expected/multi_mx_function_call_delegation_0.out

```diff
@@ -409,6 +406,7 @@ BEGIN
         WHERE id >= x
         ORDER BY 1, 2;
 END;$$;
+RESET citus.enable_metadata_sync;
```

Comment:
thinking on these cases a little more as well, I think we should be able to remove on/off switches from here as well.

I know that we have `-- before distribution ...`, but if the distribution is with a `parameter`, then creating these functions should NOT impact the behavior. Before the distribution, function is executed on the coordinator, after the distribution commands is executed on the worker.

And, removing these on/off actually makes it closer to an actual user experience. 

These on/off things scares me to avoid weird edge cases.